x/ccv/provider/migrations/v8/migrations.go

```diff
@@ -218,6 +218,12 @@ func MigrateLaunchedConsumerChains(ctx sdk.Context, store storetypes.KVStore, pk
 			return err
 		}
 
+		// chainId -> Prioritylist
+		err = rekeyChainIdAndConsAddrKey(store, providertypes.PrioritylistKeyPrefix(), chainId, consumerId)
```

Comment:
No need to add this here I think, since this is for a migration that will happen before we have the priority vals feature
@mpoke could you double-check that that's true? I'm not very on top of the ICS release schedule atm

Reply:
+1. This migration already executed (see [here](https://github.com/cosmos/interchain-security/blame/v6.0.0/x/ccv/provider/module.go)). No changes needed here.

Reply:
I removed the priority list from migrations.go and migrations_test.go.